core/base/extended_float.hpp

```diff
@@ -67,7 +67,8 @@ namespace detail {
 
 
 template <std::size_t, typename = void>
-struct uint_of_impl {};
+struct uint_of_impl {
```

Comment:
`clang-format-8` seems to disagree on these change, which version are you using?

Reply:
I tried to force `clang-format-10`, but for some reason, it wants to use `clang-format-5` in the `format` portion. Any suggestions on how to change the version used? I was thinking about removing the `clang-format` binary and replace it with a symlink, but I would prefer to not do that.

Reply:
You can replace the path in CMakeCache.txt